pkg/api/defaults.go

```diff
@@ -599,6 +599,9 @@ func (p *Properties) setMasterProfileDefaults(isUpgrade bool) {
 	// set default to VMAS for now
 	if p.MasterProfile.AvailabilityProfile == "" {
 		p.MasterProfile.AvailabilityProfile = AvailabilitySet
+		if p.IsAzureStackCloud() {
```

Comment:
Do we need this ?. As line 601 is already setting to AvailabilitySet.

Reply:
And I dont think so AKSe support VMSS on master ?.

Reply:
Not needed now because Az and AzS match at this point in time.
It will be handy if Az's default change.